mrtt-ui/src/data/questionMappingProjectDetails.js

```diff
@@ -0,0 +1,7 @@
+export const projectDetailsMapping = {
+  hasProjectEndDate: '1.1a',
+  projectStartDate: '1.1b',
+  projectEndDate: '1.1c',
+  countries: '1.2',
+  siteArea: '1.3'
+}
```

Comment:
I like this. As you create more forms you may want to add some nesting to the structure. Something like (or whatever is more suitable)

```js
export const projectDetailsMapping = {
  registration: {
    siteDetailsAndLocation: {
      hasProjectEndDate: '1.1a',
      projectStartDate: '1.1b',
      projectEndDate: '1.1c',
      countries: '1.2',
      siteArea: '1.3'
    },
    siteBackground: {
      //etc...
    }
  },
  intervention: {
    //etc...
  }
}
```

Or you could export them separately:
```js
export const registrationMapping = {
  siteDetailsAndLocation: {
      hasProjectEndDate: '1.1a',
      projectStartDate: '1.1b',
      projectEndDate: '1.1c',
      countries: '1.2',
      siteArea: '1.3'
    },
    siteBackground: {
      //etc...
    }
}

export const interventionMapping = {
  //etc...
}
```

Reply:
got the idea from Angelo! I was thinking of exporting them separately, but can work either or ! I'm open to either

Reply:
I think I prefer it as one object :) just updated